File: ruby/test/01/first_kata_test.rb
require "test/unit"
require "../../katas/01/first_kata"

class FirstKataTest < Test::Unit::TestCase
  def test_sum
    assert_equal(23, FirstKata.new.sum(9, 3, 5), "result should be 23")
    assert_equal(233168, FirstKata.new.sum(999, 3, 5), "result should be 233168")
  end
end

File: haskell/test/01/FirstKataTest.hs
import Test.HUnit
import FirstKata 

testFkSumFirst = TestCase $ assertEqual "result should be 23" 23 (fkSum 9 3 5)
testFkSumSecond = TestCase $ assertEqual "result should be 233168" 233168 (fkSum 999 3 5)

suite = TestList [ testFkSumFirst, testFkSumSecond ]
